src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -257,9 +257,33 @@ int IComparable<CompletionItem>.CompareTo(CompletionItem other)
             return result;
         }
 
-        public override string ToString()
+        public override string ToString() => DisplayText;
+
+        internal bool ShouldBeFilteredOutOfCompletionList(
+            ImmutableDictionary<CompletionItemFilter, bool> filterState)
```

Comment:
Does this function belong on this type?  It doesn't use any of the class's state.